API/API.mli

```diff
@@ -584,13 +569,11 @@ sig
     val fold_outside : (('c, 't) Declaration.pt -> 'a -> 'a) -> ('c, 't) pt -> init:'a -> 'a
 
     (** Return the set of all identifiers bound in a given named-context. *)
-    val to_vars : ('c, 't) pt -> Names.Id.Set.t
```

Comment:
OK, but I think it makes sense to remove also the accompanying comment from `API/API.mli` file.